ui/src/icons/Icons.tsx

```diff
@@ -105,32 +112,43 @@ export const MarketIcon = () => (
+    </svg>
 )
 
-export const UserIcon = () => (
-    <img
-        className="icon user-icon"
-        src={user}
-        alt="command"/>
+export const UserIcon: React.FC<IconProps> = ({ size }) => (
+    <svg className={`icon user-icon icon-size-${size}`} width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="18" height="18">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M0 0H17.9999V17.9997H0V0Z" fill="white"/>
+        </mask>
+        <g mask="url(#mask0)">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M14.7325 14.4805C14.5144 13.1707 13.7531 10.7609 11.1685 9.87462C12.096 9.20121 12.7059 8.11591 12.7059 6.88238C12.7059 4.83568 11.0467 3.1765 9.00002 3.1765C6.95331 3.1765 5.29413 4.83568 5.29413 6.88238C5.29413 8.11591 5.90402 9.20121 6.83155 9.87462C4.24696 10.7609 3.4846 13.1697 3.26755 14.4805C1.90272 13.0532 1.05884 11.1251 1.05884 9.00003C1.05884 4.62074 4.62072 1.05885 9.00002 1.05885C13.3793 1.05885 16.9412 4.62074 16.9412 9.00003C16.9412 11.1251 16.0973 13.0532 14.7325 14.4805ZM6.35319 6.88239C6.35319 5.42227 7.54119 4.23533 9.00025 4.23533C10.4593 4.23533 11.6473 5.42227 11.6473 6.88239C11.6473 8.34145 10.4593 9.52945 9.00025 9.52945C7.54119 9.52945 6.35319 8.34145 6.35319 6.88239ZM4.23522 15.3381C4.23733 15.0523 4.3411 10.5883 8.99992 10.5883C13.6355 10.5883 13.7615 15.047 13.7636 15.3381C12.4358 16.3387 10.7883 16.9412 8.99992 16.9412C7.21051 16.9412 5.56404 16.3387 4.23522 15.3381ZM9 0C4.02988 0 0 4.02882 0 9C0 13.9701 4.02988 18 9 18C13.9701 18 18 13.9701 18 9C18 4.02882 13.9701 0 9 0Z" fill="#B4F5A3"/>
+        </g>
+    </svg>
 )
 
-export const WalletIcon = () => (
-    <img
-        className="icon wallet-icon"
-        src={wallet}
-        alt="command"/>
+export const WalletIcon: React.FC<IconProps> = ({ size }) => (
+    <svg className={`icon wallet-icon icon-size-${size}`} width="14" height="12" viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M10.8109 1.40002L10.8887 1.78891H12.0554L11.822 0.622244C11.7443 0.311133 11.4332 0.0777996 11.122 0.155577L1.71094 1.78891H8.55538L10.8109 1.40002Z" fill="#B4F5A3"/>
+        <path d="M13.2219 4.43338L13.1442 2.41116C13.1442 2.10005 12.9108 1.78894 12.5219 1.78894H12.0553H10.8886H8.55527H1.71083H0.777496C0.466385 1.78894 0.155273 2.02227 0.155273 2.41116V10.8889C0.155273 11.2001 0.388607 11.5112 0.777496 11.5112H12.5997C12.9108 11.5112 13.2219 11.2778 13.2219 10.8889V9.48894C13.5331 9.48894 13.8442 9.25561 13.8442 8.86672V4.97783C13.8442 4.66672 13.5331 4.43338 13.2219 4.43338ZM11.9775 10.3445H1.39972V2.95561H12.0553V4.43338H8.08861C7.62194 4.43338 7.38861 4.66672 7.38861 4.97783V8.9445C7.38861 9.25561 7.62194 9.56672 8.01083 9.56672H11.9775V10.3445ZM12.5997 8.32227H11.9775H8.55527V5.60005H11.9775H12.5997V8.32227Z" fill="#B4F5A3"/>
+    </svg>
```

Comment:
many of these icons changed, and i imported them as svg so it is easier to apply classnames / styles